tests/test_playbooks/users_role.yml

```diff
@@ -0,0 +1,73 @@
+---
+# The following sensitive variables can be specified in the server.yml
+# ldap_host:
+# ldap_account:
+# ldap_password:
+# ldap_base_dn:
+# ldap_groups_base:
```

Comment:
If those are sensitive, you should provide dumb examples in the `server.yml.example´ and, more importantly, filter them in `vcr_python_wrapper` from being written to the `tests/test_playbooks/fixtures/users_role-*.yml` files.

Reply:
I have added examples and extended the filter in the "wrapper.py". The result/output is now good for me.

Reply:
Can you explain to me why the tests here are not successful? Mine were ok. 
I can only find a message like "can not open to record" or similar.

Reply:
Sorry for the delay. I will try to find time to dig into this.
Looking at the logs it says it found 1 similar request with no failed matcher. This does not make sense to me.

Reply:
N.p. . Me neither ^^